### ISSUE TITLE
[NuGet.org Bug]: A11y_NuGet Service_.Json_InfoSection_Reflow : On Zooming the screen to 400% "Graph section" is not visible.

### ISSUE THREAD

**Issue by joelverhagen** [NuGet/NuGetGallery#8920]
### Impact

It's more difficult to complete my work

### Describe the bug

Source: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1439345

### Repro Steps

1. Hit the URL: https://www.nuget.org/
2. Tab navigate to "Sign in" control and hit enter.
3. Tab navigate to "Sign in with Microsoft" control and hit enter.
4. Tab navigate to search box and search for a key term (say “json”).
5. Tab navigate and click on "Newtonsoft.json" control and hit enter.
6. Tab navigate to "Info" and "Statistics" section present on right of the screen.
7. Zoom the screen to 400% observe that "graph section" is not visible.

When we zoom the screen to 400% the "Graph section" present on the "Statistics" page is not visible. 

### Expected Behavior

The "Graph section" should be completely visible when we zoom the screen to 400%.

**Comment by ryuyu** [comment]
This appears to be due to the element showing example glob patterns having a larger width than the page (which has responsively reflowed to the resize). Exploring options now.
